cmd/kube-apiserver/app/server.go

```diff
@@ -348,6 +350,7 @@ func CreateKubeAPIServerConfig(
 
 			ServiceAccountIssuer:        s.ServiceAccountIssuer,
 			ServiceAccountMaxExpiration: s.ServiceAccountTokenMaxExpiration,
+			ExtendExpiry:                s.Authentication.ServiceAccounts.ExtendExpiry,
```

Comment:
nit: use `Expiration` instead of `Expiry` everywhere

Reply:
Done